frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -594,21 +614,40 @@ export function Experiment_(): JSX.Element {
                                 </Col>
                             </Row>
                         )}
-                        {showWarning && experimentResults && !areResultsSignificant && (
+                        {showWarning && experimentResults && !areResultsSignificant && !experimentData.end_date && (
                             <Row align="middle" className="not-significant-results">
-                                <Col span={19} style={{ color: '#f96132' }}>
-                                    Your results are <b>not statistically significant</b>.{' '}
-                                    {experimentData.end_date ? '' : "We don't recommend ending this experiment yet."}
-                                    {significanceTooltip && (
-                                        <Tooltip placement="right" title={significanceTooltip}>
-                                            <InfoCircleOutlined style={{ color: '#f96132', padding: '4px 2px' }} />
-                                        </Tooltip>
-                                    )}
+                                <Col span={23} style={{ color: '#2D2D2D' }}>
+                                    <b>Your results are not statistically significant</b>. {significanceBannerDetails}{' '}
+                                    {experimentData?.end_date ? '' : "We don't recommend ending this experiment yet."}{' '}
+                                    See our{' '}
+                                    <a href="https://posthog.com/docs/user-guides/experimentation#funnel-experiment-calculations">
```

Comment:
Is "#funnel-experiment-calculations" relevant for trends too?

Reply:
Just the heading below, visible on the same page, so should be ok